src/pytest_springfield/README.md

```diff
@@ -0,0 +1,6 @@
+`dottedname` Module
+===================
+
+This module exists as a dummy module to help test imports of
+entity classes when using dotted-name strings for `EntityField`
+types.
```

Comment:
Yea that's what I figured, but I didn't want to put my test garbage in springfield src and I just happened to notice that the pytest_ dir was on my python path. Should I move it somewhere else?